pkg/upload/react-native-ios.go

```diff
@@ -132,7 +132,13 @@ func ProcessReactNativeIos(
 					plistPath = plistPathExpected
 					log.Info("Found Info.plist at: " + plistPath)
 				} else {
-					log.Info("No Info.plist found at: " + plistPathExpected)
+					plistPathExpected = filepath.Join(buildSettings.ProjectTempRoot, "ArchiveIntermediates", scheme, "BuildProductsPath", filepath.Base(buildSettings.BuiltProductsDir), buildSettings.InfoPlistPath)				
+					if utils.FileExists(plistPathExpected) {
+						plistPath = plistPathExpected
+						log.Info("Found Info.plist at: " + plistPath)
```

Comment:
Thinking about this some more, we should state it was found in the "expected location" as we only drill into here when `--plist-path` is not defined.